src/utils/urlEncoding.ts

```diff
@@ -0,0 +1,47 @@
+// We cannot have a dash in the symbol because:
+// 1. We use a `-` to separate token symbols in the URL
+// 2. Tokens can have dashes in their symbols
+// 3. Even if we encode dashes to `%2D`, browsers auto convert them back to `-`
+// For these reasons, we are picking a symbol that is not ambiguous with a dash
+// and it have very little chance to be used as part of a token symbol.
+// Meet, the question circle dash ⊝ https://www.fileformat.info/info/unicode/char/229d/index.htm
```

Comment:
Brb, creating a Token `T⊝KEN`

Reply:
You are right, better not advertise that because of people like you!

:D